File: 250106/연속되는 수 4/continuous-number4.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int N, arr[1000];
    cin >> N;
    for (int i=0; i<N; i++) {
        cin >> arr[i];
    }
    int cnt=1, max=0;
    for (int i=0; i<N; i++) {
        if (i >= 1 && arr[i-1] < arr[i]) {
            cnt++;
        } else {
            cnt = 1;
        }
        if (max < cnt) max = cnt;
    }
    cout << max;
    return 0;
}

File: 241006/연속되는 수 4/continuous-number4.py
N = int(input())
arr = [int(input()) for i in range(N)]

ans, cnt = 0, 0
for i in range(N):
    if i >= 1 and arr[i] > arr[i-1]:
        cnt += 1
    else:
        cnt = 1
    ans = max(ans, cnt)

print(ans)